dex/msgjson/types.go

```diff
@@ -1270,6 +1272,9 @@ type BondAsset struct {
 
 // ConfigResult is the successful result for the ConfigRoute.
 type ConfigResult struct {
+	// Logo is an optional custom logo that can be be provided by server
+	// operators.
+	Logo []byte `json:"logo"`
```

Comment:
As I said in https://github.com/decred/dcrdex/issues/1256#issuecomment-1358479967, adding the logo bytes to the `ConfigResult` is a **no-go** for me.

The HTTP endpoint (`/logo`) that you added will serve this purpose.

I don't even think the dexc backend needs to concern itself with the logo.  The frontend not being able to request the logo because it's missing the cert is solvable by putting the cert in `core.Exchanges`.  However, if you really want the backend to be retrieving the logo, not the frontend (the only consumer of the logo), then core should fetch it via HTTP regardless of whether it's a configured exchange or not.  @buck54321 noted it must be HTTP in the issue.